services/ui-src/src/styles/_header.scss

```
@@ -101,61 +99,62 @@
 
       &.open {
         background: $brand-tertiary;
-        display: block;
-        padding: ($padding * 2);
+        display: flex;
+        flex-direction: column;
+        align-items: baseline;
+        padding: $padding;
         position: absolute;
         right: 0;
         transition: all 2s;
       }
-
-      .manage-account {
-        padding: $padding 0 $padding ($padding * 4);
+      .contact-us {
+        padding: ($padding * 2) 0 ($padding * 1.5);
         position: relative;
-
+        width: 100%;
         a {
+          padding: ($padding * 2) ($padding * 9.75) ($padding * 1.5) $padding;
           &:before {
-            content: "\f044";
+            content: "\f05a";
             font-family: "FontAwesome";
-            position: absolute;
             top: ($margin * 1.5);
             left: 0;
```

Because the icons are slightly different sizes, the `padding-right` pushes the labels just a tiny bit out of alignment. This (when applied to all three) will fix that. I've made the same suggestion on the other two.
```suggestion
            display: inline-block;
            width: 16px;
```